packages/loader/driver-utils/src/network.ts

```diff
@@ -46,14 +46,16 @@ export class GenericNetworkError extends LoggingError implements IDriverErrorBas
 export class AuthorizationError extends LoggingError implements IAuthorizationError {
     readonly errorType = DriverErrorType.authorizationError;
     readonly canRetry = false;
+    readonly claims: { tag: string, value: string | undefined }; // See ITaggedTelemetryPropertyType
```

Comment:
Why not using ITaggedTelemetryPropertyType here? Are you trying to narrow down type (string)?
undefined seems like should not be there, i.e. it should be { tag: string, value: string } | undefined and caller do a check (inconvenient, but feels more correct)

Reply:
It doesn't exist yet (see Kabir's PR), and this value is actually for runtime use not logging.  I do think I agree about `value: string` and not having `undefined` in there.